cli/README.md

```diff
@@ -20,20 +20,42 @@ uv sync
 
 ## Usage
 
-**Note** The `rabbitmq`, `export-api` and `imaging-api` services must be started prior to using the CLI
+**Note:** The `rabbitmq`, `export-api` and `imaging-api` services must be started prior to using the CLI
 This is done by spinning up the necessary Docker containers through `docker compose`.
+
+See general pixl commands and subcommands with:
+
+```bash
+pixl --help
+```
+
+### Starting PIXL
+
 For convenience, we provide the `pixl dc` command, which acts as a wrapper for `docker compose`,
 but takes care of some of the configuration for you.
 
-See the commands and subcommands with
+**1) Default Start-up**
+```bash
+pixl dc up
+```
+By default, `pixl dc up` runs with the option: `--profile postgres-exposed`. You should use this option if PIXL DB and Orthanc Raw 
+will share the same postgres instance. The `ORTHANC_RAW_DB_*` and `PIXL_DB_*` variables should have the same values, as seen in the 
+[default .env.sample](../.env.sample).
 
+**2) Start-up with External PIXL DB**
+Set the port number environment variable of your external PIXL DB:
 ```bash
-pixl --help
+CLI_PIXL_DB_PORT=7001
+```
+
+Start-up PIXL:
+```bash
+pixl dc up --profile postgres
```

Comment:
Can I suggest instead that we have an environmental variable that the CLI reads and works out if it should be exposed or not? I don't want to have to always remember when re-upping production whether it should be exposed or not.

Maybe?
```
EXTERNAL_PIXL_DB=TRUE
```

Reply:
oh that's a much better idea